search/run_search_engine.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python2.7
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+import argparse
+import os
+import search_engine_pylib as search
+
+
+DIR = os.path.dirname(__file__)
+RESOURCE_PATH = os.path.realpath(os.path.join(DIR, '..', 'data'))
+MWM_PATH = os.path.realpath(os.path.join(DIR, '..', 'data'))
+
+parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
```

Comment:
maybe move cmd parsing to a separate function? or even some module with agile initialization for another implementations of a run script?

Also most of the time it is a good idea to write run-time code (not import-time) and functions (not module-scoped code snippets)